src/Member.hpp

```diff
@@ -9,21 +9,21 @@
 
 class Member
 {
-public:
-    explicit Member (Board& board_);
+    public:
+        explicit Member (Board& board_);
 
-    bool isPositionAlreadyCalculated( const std::pair<int, int> position );
-    std::vector<std::pair<int, int> > &&getSuccessfulSteps();
+        bool isPositionAlreadyCalculated( const std::pair<int, int> position );
+        std::vector<std::pair<int, int> >&& getSuccessfulSteps();
```

Comment:
At other places you writes reference char at right side. You must choose final variant.
Thanks.